### PR TITLE
Update README with basic faq

### PR DESCRIPTION
**Pull request by tiwiz**
Hello!

I thought it was useful to add this basic F.A.Q. section, I think it could help some people

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,13 @@ design as well as avoiding complexity in the generated code. This means that all
 be declared in the primary constructor of the class. This means that `var` properties declared outside the constructor will
 not be serialized.
 
+F.A.Q.
+---
+
+*Q: I can't generate `KotshiJsonAdapterFactory`-annotated class*
+
+**A: make sure to add the `kotlin-kapt` plugin in the `build.gradle` file of your project.**
```
Comment (ansman) [inline README.md]: You actually don't need to apply that plugin to make it work so this comment is a bit misleading.

Comment (tiwiz) [reply README.md]: I couldn't get it generated otherwise, on Android Studio 3.0.1... could it be a bug?

Comment (ansman) [reply README.md]: Hmm, strange. I'll have to investigate a bit but I thought you only needed to apply the plugin if you wanted to use kapt3

Comment (tiwiz) [reply README.md]: I have no idea, honestly. Just started a new project from scratch, and without the plugin it was not working. I can try with 3.1 as well :-)

Comment (ansman) [reply README.md]: Well, which version of AS you are using does not matter. The build tools version and kotlin version could perhaps affect it.

Comment (tiwiz) [reply README.md]: Right. I'm on Kotlin 1.2.21 and tools 3.0.1, with gradle 4.1

**Comment by ansman** [comment]
Closing due to inactivity

**Pull request closed without merging**